dictionaries/software-terms/dict/softwareTerms.txt

```diff
@@ -1,6 +1,15 @@
 
 # cspell-tools: keep-case no-split
 
```

Comment:
Maybe you could add here and in all generated files something like

```
# GENERATED FILE, DO NOT EDIT
# this file is the concatenation of foo.txt, bar.txt
# edit them if needed
# please note comments are removed in this file, you might get more context in those files
```

Something like that.

Unless I missed something

Reply:
I'm unsure if you saw this @jason3s

Reply:
You may think I'm micromanaging you 😅

But maybe a simple emoji reaction on my comment could help me to understand you saw it. It avoid posting a comment when you have nothing to add

Reply:
Good point. I would need to change the dictionary compiler for that. There is (or should be) a comment to that effect in the [`dict/README.md`](https://github.com/streetsidesoftware/cspell-dicts/blob/main/dictionaries/software-terms/dict/README.md)

It is not 100% necessary to check these file in, but it is very helpful for detecting unexpected changes.

Reply:
I raised the point because I started reviewing the first file and I was thinking

I will report him these new words could have a comment as we talked. So I started posting comments in pending mode.

Then when I was done with first file, I was like … I reviewed a generated file expunged from its comments.

So I commented about the file, and the possible headers

Reply:
Good point, I have marked it as auto generated. See: https://docs.github.com/en/repositories/working-with-files/managing-files/customizing-how-changed-files-appear-on-github